streamer/src/nonblocking/quic.rs

```diff
@@ -116,6 +117,21 @@ pub async fn run_server(
     }
 }
 
+fn prune_unstaked_connection_table(
+    unstaked_connection_table: &mut MutexGuard<ConnectionTable>,
+    max_unstaked_connections: usize,
+    stats: Arc<StreamStats>,
```

Comment:
nit: i think we can take `&StreamStats` here? so, we can skip `Arc::clone()` at caller side.